dependencies.props

```diff
@@ -28,17 +28,13 @@
   <PropertyGroup>
     <CoreClrCurrentRef>f31086500dc58cefe7b1275300f6fc07b38c1bcc</CoreClrCurrentRef>
     <BuildToolsCurrentRef>fc12d73e8ea045517fde9b4bd1c89065917a5413</BuildToolsCurrentRef>
-    <PgoDataCurrentRef>fc12d73e8ea045517fde9b4bd1c89065917a5413</PgoDataCurrentRef>
-    <IbcDataCurrentRef>fc12d73e8ea045517fde9b4bd1c89065917a5413</IbcDataCurrentRef>
   </PropertyGroup>
 
   <!-- Tests/infrastructure dependency versions. -->
   <PropertyGroup>
     <PgoDataPackageVersion>99.99.99-master-20190208.1</PgoDataPackageVersion>
     <MicrosoftNETCoreRuntimeCoreCLRPackageVersion>3.0.0-preview-27316-02</MicrosoftNETCoreRuntimeCoreCLRPackageVersion>
     <XunitPackageVersion>2.4.1</XunitPackageVersion>
-    <IbcDataPackageVersion>99.99.99-master-20190208.1</IbcDataPackageVersion>
-    <IbcMergePackageVersion>4.6.0-alpha-00001</IbcMergePackageVersion>
```

Comment:
We need to add this line back in, I think. The IBCMerge package is separate and doesn't come from darc.

Reply:
Is it a fixed version?

Reply:
Or rather, does it build using the old system?

Reply:
It is a fixed version, not actively being worked on. It builds manually and a package is uploaded manually.

Reply:
Ok, I don't need to revert the update steps then. We should eventually specify this as a toolset dependency in the new format, but I'll leave that for another PR.

Reply:
Makes sense